src/blog-utils/StaticAddonCard/index.js

```diff
@@ -0,0 +1,53 @@
+/* @flow */
+import * as React from 'react';
+
+import { getAddonIconUrl } from 'amo/imageUtils';
+import { nl2br, sanitizeHTML } from 'amo/utils';
+import AddonBadges from 'amo/components/AddonBadges';
+import AddonTitle from 'amo/components/AddonTitle';
+import GetFirefoxButton, {
+  GET_FIREFOX_BUTTON_TYPE_ADDON,
+} from 'amo/components/GetFirefoxButton';
+import type { AddonType } from 'amo/types/addons';
+
+import './styles.scss';
+
+type Props = {|
+  addon: AddonType,
+|};
+
+const StaticAddonCard = ({ addon }: Props): React.Node => {
+  if (!addon) {
+    return null;
+  }
+
+  const summary = addon.summary ? addon.summary : addon.description;
```

Comment:
Assuming there isn't some switching around in the frontend reducer, etc, `summary` is the field we extract from the manifest - and force to have a value before a version can be submitted for listed approval - while `description` is optional, so it's unlikely an add-on will have a description but not a summary.

Reply:
Interesting, I wonder why we have that on AMO then. I will file an issue to investigate.

Reply:
It looks like we're doing this for themes: https://github.com/mozilla/addons-frontend/blob/cabfc342cdbd1fed6c9907051cdedceb8998971a/src/amo/pages/Addon/index.js#L422-L424

Reply:
iirc that was a LWT thing.  Static themes follow the same rules as extensions.